.golangci.yaml

```diff
@@ -86,6 +86,7 @@ linters:
       - third_party$
       - builtin$
       - examples$
+      - lxd/idmap/idmapset_linux\.go$
```

Comment:
this is fine if its short term, but i worry we will forget to look at it again.

Reply:
I've been avoid touching this file as it's full of linter problems. I intend to eventually get to it but I'll need to carve out some time for it.